Makefile

```diff
@@ -10,10 +10,7 @@ MANINSTALLDIR=${PREFIX}/share/man
 GO_MD2MAN ?= /usr/bin/go-md2man
 
 ifeq ($(DEBUG), 1)
-  GOGCFLAGS += "-N -l"
-else
-  GOGCFLAGS ?= ""
-  DEBUG ?= 0
```

Comment:
Doesn't break anything, but you still get `DEBUG=` below on the call to make in the container. I thought it was a bit cleaner to say DEBUG=0 in the false case.